scylla-cql/Cargo.toml

```diff
@@ -14,33 +14,33 @@ scylla-macros = { version = "0.4.0", path = "../scylla-macros" }
 byteorder = "1.3.4"
 bytes = "1.0.1"
 tokio = { version = "1.12", features = ["io-util", "time"] }
-secrecy = { version = "0.7.0", optional = true }
+secrecy-08 = { package = "secrecy", version = "0.8", optional = true }
 snap = "1.0"
 uuid = "1.0"
 thiserror = "1.0"
 num-bigint-03 = { package = "num-bigint", version = "0.3", optional = true }
 num-bigint-04 = { package = "num-bigint", version = "0.4", optional = true }
 bigdecimal-04 = { package = "bigdecimal", version = "0.4", optional = true }
-chrono = { version = "0.4.27", default-features = false, optional = true }
+chrono-04 = { package = "chrono", version = "0.4", default-features = false, optional = true }
```

Comment:
It's the same thing here, please check that scylla-cql really doesn't require any version higher that `0.4.0`